tests/v2/validation/publicapi/clusterroletemplatebindings/crtb_create_steve_test.go

```diff
@@ -0,0 +1,177 @@
+	}
+	for _, rtn := range tests {
+		crtbName := namegen.RandStringLower(crtbNameWithTenChar)
+		jsonData := clusterRoleTemplateBindingTemplate(crtb.clusterID, crtbName, crtb.newUser.ID, rtn.roleTemplateName)
+
+		resp, err := client.Steve.SteveType(crtbApiEndPoint).Create(jsonData)
+		require.NoError(crtb.T(), err)
+		assert.Contains(crtb.T(), (*resp).JSONResp["id"], crtbName)
+		assert.Contains(crtb.T(), (*resp).JSONResp["clusterName"], crtb.clusterID)
+		assert.Contains(crtb.T(), (*resp).JSONResp["userName"], crtb.newUser.ID)
+		assert.Contains(crtb.T(), (*resp).JSONResp["roleTemplateName"], rtn.roleTemplateName)
+	}
+}
+
+func (crtb *CRTBGenTestSuite) TestCreateCrtbWithUserPrincipalNameAsInput() {
+	client := crtb.client
+	crtbName := namegen.RandStringLower(crtbNameWithTenChar)
+
+	jsonData := clusterRoleTemplateBindingTemplate(crtb.clusterID, crtbName, crtb.newUser.ID, clusterMember)
+	jsonData.UserPrincipalName = "local://" + crtb.newUser.ID
+
+	resp, err := client.Steve.SteveType(crtbApiEndPoint).Create(jsonData)
+	require.NoError(crtb.T(), err)
+
+	assert.Contains(crtb.T(), (*resp).JSONResp["id"], crtbName)
+	assert.Contains(crtb.T(), (*resp).JSONResp["clusterName"], crtb.clusterID)
+	assert.Contains(crtb.T(), (*resp).JSONResp["userName"], crtb.newUser.ID)
+	assert.Contains(crtb.T(), (*resp).JSONResp["roleTemplateName"], clusterMember)
+	assert.Equal(crtb.T(), (*resp).JSONResp["userPrincipalName"], jsonData.UserPrincipalName)
+}
+
+func (crtb *CRTBGenTestSuite) TestCreateCrtbAndInputLabels() {
+	client := crtb.client
+	crtbName := namegen.RandStringLower(crtbNameWithTenChar)
+
+	jsonData := clusterRoleTemplateBindingTemplate(crtb.clusterID, crtbName, crtb.newUser.ID, clusterMember)
+	jsonData.ObjectMeta.Labels = map[string]string{"Hello": "World"}
+
+	resp, err := client.Steve.SteveType(crtbApiEndPoint).Create(jsonData)
+	require.NoError(crtb.T(), err)
+
+	assert.Contains(crtb.T(), (*resp).JSONResp["id"], crtbName)
+	assert.Contains(crtb.T(), (*resp).JSONResp["clusterName"], crtb.clusterID)
+	assert.Contains(crtb.T(), (*resp).JSONResp["userName"], crtb.newUser.ID)
+	assert.Contains(crtb.T(), (*resp).JSONResp["roleTemplateName"], clusterMember)
+	assert.NotContains(crtb.T(), (*resp).JSONResp, jsonData.Labels)
+}
+
+func (crtb *CRTBGenTestSuite) TestValidateErrorAlreadyCrtbExisted() {
+	client := crtb.client
+	crtbName := namegen.RandStringLower(crtbNameWithTenChar)
+
+	jsonData := clusterRoleTemplateBindingTemplate(crtb.clusterID, crtbName, crtb.newUser.ID, clusterMember)
+
+	resp, err := client.Steve.SteveType(crtbApiEndPoint).Create(jsonData)
+	require.NoError(crtb.T(), err)
+	assert.Contains(crtb.T(), (*resp).JSONResp["id"], crtbName)
+
+	_, err = client.Steve.SteveType(crtbApiEndPoint).Create(jsonData)
+	require.Error(crtb.T(), err)
+	require.ErrorContains(crtb.T(), err, crtbConflictError)
+	require.ErrorContains(crtb.T(), err, crtbAlreadyExisted)
+}
+
+func (crtb *CRTBGenTestSuite) TestValidateErrorRoleTemplateNameNotExisted() {
```

Comment:
confirming that these don't depend on being ran in any particular order? 

side note; can the name be updated to `...NotExisting` or `...DoesNotExist` instead of `existed`?